mstranslator.py

```diff
@@ -3,6 +3,7 @@
 import requests
 import datetime
 import json
+import logging
```

Comment:
Why is this module imported, but not used?

Reply:
I forgot to remove, already done it.